cfme/tests/control/test_compliance.py

```diff
@@ -73,10 +60,25 @@ def assign_policy_for_testing(policy_for_testing, host, policy_profile_name):
     host.unassign_policy_profiles(policy_profile_name)
 
 
+@pytest.fixture(scope="module")
+def vddk_url(provider):
+    try:
+        major, minor = provider.version.split(".")
```

Comment:
This has hit an AttributeError exception in PRT, since provider.version is returning an `int`, which doesn't have a `split()` method.